### PR TITLE
Brazilian Portuguese Translation - Review + Update to 1.5

### PR DESCRIPTION
**Pull request by pedrobbs00**
Finally got around to updating the translation + reviewing and correcting a lot of mistakes and counterintuitive naming.

### CODE REVIEW

**Comment by manongjohn** [comment]
@pedrobbs00 Better late than never!  I'll include it in the next fix release (Feb).

You may also wish to translate the new tips file `stuff\doc\tips.md`.  If you do, create and place it in folder `stuff\doc\Português(Brasil)\`

Be careful which editor you use as it needs to be able to save in UTF-8 encoding.  Use `Notepad++`

**Comment by pedrobbs00** [comment]
Thanks, @manongjohn! Can't wait! 😄

And perfect! I just tried it out and seems to be working perfectly (I used Notepad++ just to be safe and it seems to be working correctly). But I just noticed, interestingly enough, the "Tips..." text seems to be the only thing left to translate, although I couldn't find it in any .ts file.

image

File: stuff/doc/Português((Brasil)/tips.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  - Clique com o botão direito do mouse sobre qualquer Célula e selecione `Habilitar/Desabilitar Holds Implícitos` para mudar os modos.
+  - Converta Cenas existentes de/para usar Holds Implícitos com `Cena` -> `Converter para Holds Implícitos/Explícitos`.
+- Algumas funções e configurações adicionais do OpenToonz estão escondidas.
+  - Habilite-as em `Preferências` -> `Geral` -> `Mostrar Preferências e Opções Avançadas*` e reinicie o Tahoma2D.
+
+-----
+## Dicas
+- Animações e recursos externos (desenhos) de uma Cena são salvos em arquivos separados dentro de uma pasta de Projeto.
+- Crie um Projeto `Tahoma2D` em sua pasta Documentos e use como seu projeto geral ao invés do padrão `sandbox`.
+- Use `Save All`, regularly, to save both scene and asset files. `Save Scene` does not save asset changes, just the scene changes.
```
Comment (manongjohn) [inline stuff/doc/Português((Brasil)/tips.md:37]: This line needs to be removed.

Comment (pedrobbs00) [reply stuff/doc/Português((Brasil)/tips.md]: Corrected! I was copying and then translating, by mistake left one of the originals in there xD

**Comment by manongjohn** [comment]
@pedrobbs00 Thanks for updating the translations!

**Pull request merged**